src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -169,29 +169,27 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
 
         # invoke enable-NestedHyperV.ps1 again to attach Disk to Nested
         if enable_nested:
-            logger.info("Running hyperv")
+            logger.info("Running Script win-enable-nested-hyperv.ps1 to install HyperV")
```

Comment:
Is win-enable-nested-hyperv.ps1 executed?

Reply:
yes